.travis.yml

```diff
@@ -1,8 +1,13 @@
 language: ruby
-sudo: false
+sudo: true
 rvm:
   - 2.3.1
   - 2.2.4
   - ruby-head
 services:
-  - rabbitmq
+  - docker
+before_install:
+  - docker-compose up -d
```

Comment:
It seems easier (to me) to specify what I need via docker compse, which will stand up Rabbit and Redis.